wayland-display-core/src/comp/mod.rs

```diff
@@ -331,16 +342,16 @@ pub(crate) fn init(
                         .to_logical(output.current_scale().fractional_scale())
                         .to_i32_round();
                     for window in data.state.space.elements() {
-                        let toplevel = window.toplevel();
+                        let toplevel = window.toplevel().unwrap();
                         let max_size = Rectangle::from_loc_and_size(
                             (0, 0),
                             with_states(toplevel.wl_surface(), |states| {
                                 states
                                     .data_map
                                     .get::<XdgToplevelSurfaceData>()
-                                    .map(|attrs| attrs.lock().unwrap().max_size)
+                                    .map(|attrs| todo!("Used to be attrs.lock().unwrap().max_size but max_size is now not present!"))
```

Comment:
`max_size` is now missing, what am I missing here?

Reply:
```
states.cached_state.current::<SurfaceCachedState>().max_size
```